Plugin Cubot/src/main/java/net/simon987/cubotplugin/event/DeathCountListener.java

```diff
@@ -0,0 +1,32 @@
+package net.simon987.cubotplugin.event;
+
+import net.simon987.server.event.GameEvent;
+import net.simon987.server.event.GameEventListener;
+import net.simon987.server.game.objects.ControllableUnit;
+import net.simon987.server.game.objects.GameObject;
+import net.simon987.server.logging.LogManager;
+
+public class DeathCountListener implements GameEventListener {

```

Comment:
Should be `DeathListener`. You're listening for deaths, not death counts

Reply:
done